airflow/migrations/versions/0047_3_0_0_add_dag_versioning.py

```diff
@@ -141,7 +141,7 @@ def upgrade():
         batch_op.add_column(sa.Column("created_at", UtcDateTime(), nullable=False, default=timezone.utcnow))
 
     with op.batch_alter_table("task_instance", schema=None) as batch_op:
-        batch_op.add_column(sa.Column("dag_version_id", UUIDType(binary=False)))
+        batch_op.add_column(sa.Column("dag_version_id", UUIDType(binary=False), nullable=False))
```

Comment:
What happens to old tis created before this column is added? I think we need some data migration work here.

Reply:
Hmm. Yeah, quite needed